cleanup/tests/test_daemon.py

```
@@ -0,0 +1,392 @@
+    raise RuntimeError("daemon socket never appeared")
+
+
+def _shutdown(socket_path: str, thread: threading.Thread) -> None:
+    """Send SIGTERM-equivalent: close socket via direct unlink + thread death."""
+    # The cleanest cross-thread shutdown is to send a signal at process level,
+    # but tests share a process. Instead we send a malformed connection and
+    # then unlink the socket to break the accept loop.
+    try:
+        os.kill(os.getpid(), 0)  # noop; kept for clarity
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)